gst.so/src/gst_worker.cpp

```diff
@@ -64,6 +65,19 @@ using std::string;
 
 bool GSTWorker::bjson = false;
 
+// common util
+void *json_node_create(std::string module_name, std::string action_name,
```

Comment:
Can we move this function to rvs util or separate json util file ?

Reply:
in the updated files, it is already in rvs_util